tests/Controller/CRUDControllerTest.php

```diff
@@ -587,7 +587,7 @@ public function testRender()
         $this->parameters = [];
         $this->assertInstanceOf(
             'Symfony\Component\HttpFoundation\Response',
-            $this->controller->render('FooAdminBundle::foo.html.twig', [], null)
```

Comment:
shouldn't we keep them as legacy tests?

Reply:
Uuuh yeah. Please copy that from my failed `__call` PR (it's the `@expectedDeprecation` annotated part)